marketplace/commercelayer-sku/index.html

```diff
@@ -0,0 +1,239 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Commerce Layer UI Extension</title>
+    <link rel="stylesheet" href="https://contentful.github.io/ui-extensions-sdk/cf-extension.css">
```

Comment:
@Shy All these packages should be added locally as npm dependencies to let the bundler do its job